### ISSUE TITLE
ContextualMenu accessibility - lists are not structured correctly

### ISSUE THREAD

**Issue by haonliu** [OfficeDev/office-ui-fabric-react#7726]
### Environment Information 
 
- __Package version(s)__: 6.127.0
- __Browser and OS versions__: win10 chrome
### Please provide a reproduction of the bug in a codepen:
 https://codepen.io/anon/pen/yGmBpJ
#### Actual behavior:
the native role of the li items are being suppressed by using role to presentation. Hence the li elments are ignored by the screen readers and their child menuitems are exposed to the Accessibility tree as children of a list instead of listitems inside a list.
This breaks the rule: ul should only have li as their direct children.

#### Expected behavior:
ul should only have li as their direct children.

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? Yes
 
Requested priority: High
 
Products/sites affected: (if applicable)

**Comment by dzearing** [comment]
Having a difficult time what component you're referring to. Can you add more details on component and maybe a codepen?

**Comment by haonliu** [comment]
the component is the contextual menu as shown in this codePen: https://codepen.io/anon/pen/yGmBpJ

**Comment by KevinTCoughlin** [comment]
My understanding of @haonliu's report is that the following DOM:

![image](https://user-images.githubusercontent.com/706967/51421215-fcd9a880-1b4f-11e9-8ca2-a9e8c64ef43e.png)

results in an accessibility tree which omits `role="listitem"` elements under the parent `role="list"`:

![image](https://user-images.githubusercontent.com/706967/51421222-1975e080-1b50-11e9-96a2-15a8ccb82fe5.png)

The `role="menuitem"` elements are the buttons within the `<li>` elements.

However, I'm not sure this breaks the following rule:

>This breaks the rule: ul should only have li as their direct children.

But _do_ think it breaks this ARIA rule:

>Lists contain children whose role is listitem, or elements whose role is group which in turn contains children whose role is listitem.

https://www.w3.org/WAI/PF/aria/roles#list

It seems like the `<li>` elements should have `role="listitem"` rather than `role="presentation"`. I wonder why `role="presentation"` was added.

**Comment by mikewheaton** [comment]
> Since this component is more a menu than it is a list... would it make more sense to mark the `<ul>` as role="menu" so that the hierarchy was menu > menuitem?

This makes the most sense to me, too.
